packages/hoppscotch-app/helpers/functional/record.ts

```diff
@@ -1,8 +1,12 @@
+import * as RNEA from "fp-ts/ReadonlyNonEmptyArray"
+
 export const tupleToRecord = <
   KeyType extends string | number | symbol,
   ValueType
 >(
-  tuples: [KeyType, ValueType][]
+  tuples:
+    | [KeyType, ValueType][]
+    | RNEA.ReadonlyNonEmptyArray<[KeyType, ValueType]>
```

Comment:
An RNEA is also an array, hence I think this type union is redudant. Is there any TypeScript contract which is broken when we remove this ?